lib/process-services-cloud/src/lib/task/services/task-cloud.service.ts

```diff
@@ -80,6 +80,16 @@ export class TaskCloudService extends BaseCloudService {
         return taskDetails && taskDetails.assignee && taskDetails.assignee === currentUser && taskDetails.isAssigned();
     }
 
+    /**
+     * Validate if a task is editable.
+     * @param taskDetails task details object
+     * @returns Boolean value if the task is editable
+     */
+    isTaskEditable(taskDetails: TaskDetailsCloudModel): boolean {
```

Comment:
isTaskEditable has the same code of canCompleteTask.
Probably is better to have a common method something like isAssignedToMe? 
```
 isAssignedToMe(assignee: string): boolean {
        const currentUser = this.identityUserService.getCurrentUserInfo().username;
        return assignee === currentUser;
    }

isTaskEditable(taskDetails: TaskDetailsCloudModel): boolean {
taskDetails && taskDetails.isAssigned() && isAssignedToMe
}
```